src/utils/getLinkFromText.tsx

```diff
@@ -15,6 +16,10 @@ export const getLinkFromText = (text: string) => {
               : `https://${word}`
           }
           key={index}
+          target={
+            isExternalUrl(word) ? externalUrlAdditionalProps.target : undefined
+          }
+          rel={isExternalUrl(word) ? externalUrlAdditionalProps.rel : undefined}
```

Comment:
You can do this on a single line between ln5-6, outside of the JSX. Just assign it to a `const`. Then you can use the react 'spread props' syntax here.

```suggestion
          {...fooConstNameYouPicked}
```

Further Reading:

- https://reactjs.org/docs/jsx-in-depth.html#spread-attributes